PubNub/Data/Managers/PNSubscriber.m

```diff
@@ -859,8 +859,12 @@ - (void)handleSubscriptionStatus:(PNSubscribeStatus *)status {
     if (!status.isError && status.category != PNCancelledCategory) {
         
         [self handleSuccessSubscriptionStatus:status];
+#ifdef PGDROID
+        // 2016/02/16 : this PG addition is causing crashes on Android ... Sending an instance of PNErrorStatus to -handleSuccessSubscriptionStatus: is not valid since it references properties specific to PNSubscribeStatus ... unsure why we're not seeing this in crash groups on iOS ...  LONG TERM solution is to probably revert both of the commits around this and roll forward to the latest PubNub and run full QA ;-)
```

Comment:
hey @mjk3979, can you take a look at this commit?
PNSubscribeStatus is the subclass of PNErrorStatus and the input parameter in the declaration of this method is PNSubscribeStatus. So I think the problem described in the comment should not happen. Or it would crash on handleSubscriptionStatus method rather than till handleSuccessSubscriptionStatus.